src/components/builds/ConfigurationWithIssues.tsx

```diff
@@ -53,9 +53,13 @@ function ConfigurationWithIssues(props: Props) {
   }
 
   // Sort issues by (line, column) and store them in a map for faster access
-  var sortedIssues = build.parsingResult.issues.slice().sort(function (left, right) {
-    return left.line - right.line || left.column - right.column;
-  });
+  var sortedIssues = build.parsingResult.issues
+    .filter(issue => {
+      return issue.path.endsWith('.cirrus.yml') || issue.path.endsWith('.cirrus.yaml');
```

Comment:
should an empty string also be treated as YAML here? just for backward compatibility